javascript/jQuerry.js

```diff
@@ -0,0 +1,122 @@
+
+var stack = [];
+
+function isNumeric(num){
+  return !isNaN(num)
+}
+
+function operation(num1, op, num2) {
+  num1 = +num1;
+  num2 = +num2;
+  if( op == "+" )
```

Comment:
The correct styling is `if (op == "+")`.
Also,
 - `===` is preferred to `==`
 - `'+'` is preferred to `"+"`
 - `if ('+' === op)` is preferred. It is better to have the expected value first in the condition
 - Always use braces